### PR TITLE
Create_plant.py

### PR DESCRIPTION
**Pull request by kosakanishkdev**
plant trees

### CODE REVIEW

File: python/plant [hunk 1]
```diff
@@ -0,0 +1,9 @@
+afforest == 1
+deforest == 0
```
Comment (tasnimzotder) [inline python/plant]: hey @kosakanishkdev here we got some error -
```suggestion
afforest = 1
deforest = 0
```

**Review by tasnimzotder — CHANGES_REQUESTED** [review]
@kosakanishkdev Can you please change `python/plant` to `python/plant.py` that is adding the `.py` extension.

**Comment by kosakanishkdev** [comment]
got up with the changes

**Comment by tasnimzotder** [comment]
That's good @kosakanishkdev.


Just one more work to do -
Change `python/plant` to `python/plant.py`

**Review by tasnimzotder — APPROVED** [review]

**Pull request merged**